dev/Materials/Acrylic/AcrylicBrush_19h1_themeresources.xaml

```diff
@@ -0,0 +1,80 @@
+            <local:AcrylicBrush
+                x:Key="SystemControlTransientAcrylicElementBrush"
+                TintColor="#2C2C2C"
+                TintOpacity="0.15"
+                TintLuminosityOpacity="0.96"
+                FallbackColor="#2C2C2C"
+                BackgroundSource="Backdrop"/>
+            <local:AcrylicBrush
+                x:Key="SystemControlTransientBackgroundInverseBrush"
+                TintColor="#FCFCFC"
+                TintOpacity="0.0"
+                TintLuminosityOpacity="0.85"
+                FallbackColor="#FCFCFC"
+                BackgroundSource="HostBackdrop"/>
+            <local:AcrylicBrush
+                x:Key="SystemControlBaseAcrylicBrush"
+                TintColor="#202020"
+                TintOpacity="0.0"
+                TintLuminosityOpacity="0.96"
+                FallbackColor="#202020"
+                BackgroundSource="HostBackdrop"/>
+            <SolidColorBrush x:Key="SystemControlDefaultBrighteningBrush" Color="#FFFFFF" Opacity="0.419"/>
+        </ResourceDictionary>
+
+        <ResourceDictionary x:Key="Light">
+            <local:AcrylicBrush
+                x:Key="SystemControlTransientBackgroundBrush"
+                TintColor="#FCFCFC"
+                TintOpacity="0.0"
+                TintLuminosityOpacity="0.85"
+                FallbackColor="#FCFCFC"
+                BackgroundSource="HostBackdrop"/>
+            <local:AcrylicBrush
+                x:Key="SystemControlTransientAcrylicElementBrush"
+                TintColor="#FCFCFC"
+                TintOpacity="0.0"
+                TintLuminosityOpacity="0.85"
+                FallbackColor="#FCFCFC"
+                BackgroundSource="Backdrop"/>
+            <local:AcrylicBrush
+                x:Key="SystemControlTransientBackgroundInverseBrush"
+                TintColor="#2C2C2C"
+                TintOpacity="0.15"
+                FallbackColor="#2C2C2C"
+                TintLuminosityOpacity="0.96"
+                BackgroundSource="HostBackdrop"/>
+            <local:AcrylicBrush
+                x:Key="SystemControlBaseAcrylicBrush"
+                TintColor="#F3F3F3"
+                TintOpacity="0.0"
+                TintLuminosityOpacity="0.9"
+                FallbackColor="#F3F3F3"
+                BackgroundSource="HostBackdrop"/>
+            <SolidColorBrush x:Key="SystemControlDefaultBrighteningBrush" Color="#FFFFFF" Opacity="0.5"/>
+        </ResourceDictionary>
+
+        <ResourceDictionary x:Key="HighContrast">
+            <StaticResource x:Key="SystemControlTransientBackgroundBrush" ResourceKey="SystemChromeMediumLowColor" />
+            <StaticResource x:Key="SystemControlTransientAcrylicElementBrush" ResourceKey="SystemChromeMediumLowColor" />
+            <StaticResource x:Key="SystemControlTransientBackgroundInverseBrush" ResourceKey="SystemBaseMediumHighColor" />
+            <StaticResource x:Key="SystemControlBaseAcrylicBrush" ResourceKey="SystemChromeMediumLowColor" />
```

Comment:
@teaP this is a question for you. I see that in our existing RS2 file, we set
```xaml
<StaticResource x:Key="SystemControlTransientBackgroundBrush" ResourceKey="SystemChromeMediumLowColor" />
```
This surprised me, as I expected a direct reference to one of the 8 HC colors, instead of added indirection. Should these HC definitions follow the other pattern in the same RS2 file?
```xaml
<SolidColorBrush x:Key="SystemControlAltHighAcrylicElementBrush" Color="{ThemeResource SystemColorWindowColor}" />
```

Reply:
Based on our conversation earlier today, I think we want to be as direct as possible, so reference the HC colors directly instead of going through SystemChromeMediumLowColor or similar.

Reply:
I've updated the all resources modified by the feature proposal and also updated the `SystemControlTransientBackgroundBrush` as it also didn't reference a HC brush.